camel-core/src/main/java/org/apache/camel/component/seda/SedaComponent.java

```diff
@@ -39,8 +39,8 @@
 
     @Metadata(label = "consumer", defaultValue = "1")
     protected int concurrentConsumers = 1;
-    @Metadata(label = "advanced")
-    protected int queueSize;
+    @Metadata(label = "advanced", defaultValue = "1000")
+    protected int queueSize = 1000;
```

Comment:
It would be nice to keep default value as a constant for runtime and tests as well. If we will decide to change this value, it will be in one place instead of multiple.

Reply:
which class can have default value defined as constant ?  Is 'EndpointHelper' class right candidate

Reply:
I can see, that there are a lot of constants in seda hardcoded on the component and endpoint levels. @davsclaus what do you think about to move constants to the separate SedaConstants class/interface?

Reply:
Yeah sure that is a good idea